src/github/github.controller.ts

```diff
@@ -0,0 +1,225 @@
+import {
+  Controller,
+  Query,
+  Get,
+  HttpStatus,
+  Post,
+  UseGuards,
+  HttpException,
+  Req,
+  Put,
+  Param,
+  Body,
+  Delete,
+} from '@nestjs/common';
+import { GithubService } from './github.service';
+import { ApiBearerAuth, ApiTags } from '@nestjs/swagger';
+import { Env } from 'src/env';
+import { JwtAuthGuard } from 'src/auth/jwt-auth.guard';
+import { UpdatePracticesDto } from './dto/update.practices.dto';
+import { TRequest } from 'src/common/types/user.types';
+import { GetApiQueryParamsSwaggerWithOrganizationName } from 'src/decorator/decorator';
+
+@Controller('github')
+@ApiTags('Github')
+export class GithubController {
+  constructor(private readonly githubService: GithubService) {}
+
+  @ApiBearerAuth()
+  @GetApiQueryParamsSwaggerWithOrganizationName()
```

Comment:
PullProofAI

The decorator `@GetApiQueryParamsSwaggerWithOrganizationName()` is referenced but not imported in the file. Please ensure the necessary import statement is added.
```diff
+ import { GetApiQueryParamsSwaggerWithOrganizationName } from 'src/decorator/decorator';
```



<!-- This is an auto-generated comment by PullProofAI -->